commands/SensorMultiLevel.go

```diff
@@ -46,7 +46,8 @@ const (
 	Moisture
 )
 
-type CmdSensorMultiLevel struct {
+type SensorMultiLevelReport struct {
+	*report
 	Type       byte
```

Comment:
This mix of public and private fields is a little weird, since users can't construct these by hand anyway, and probably shouldn't mutate these fields.

I'd just go all private and have accessor funcs for Type, Size, Scale.  And should Type have a type, rather than byte?